pkg/connectivity/pod-to-pod.go

```diff
@@ -16,29 +16,22 @@ func PodToPod(fromPod *v1.Pod, toPod *v1.Pod) common.Result {
 	// TODO
 	meshName := common.MeshName("osm")
 
-	sourcePod := k8s.Pod{
-		Namespace: k8s.Namespace(fromPod.Namespace),
-		Name:      fromPod.Name,
-	}
-
-	destinationPod := k8s.Pod{
-		Namespace: k8s.Namespace(toPod.Namespace),
-		Name:      toPod.Name,
-	}
-
 	client, err := kuberneteshelper.GetKubeClient()
 	if err != nil {
 		log.Err(err).Msg("Error creating Kubernetes client")
 	}
 
+	sourceNamespace := k8s.Namespace(fromPod.Namespace)
+	destinationNamespace := k8s.Namespace(toPod.Namespace)
+
 	outcomes := common.Run(
 		// Check source Pod's namespace
-		namespace.IsInjectEnabled(client, sourcePod.Namespace),
-		namespace.IsMonitoredBy(client, sourcePod.Namespace, meshName),
+		namespace.IsInjectEnabled(client, sourceNamespace),
```

Comment:
would you also need to change the type here? https://github.com/openservicemesh/osm-health/blob/e9c5e8e2d60c301129ce5369a1b0ae356b3bbf84/pkg/kubernetes/namespace/injected.go#L23

and here:
https://github.com/openservicemesh/osm-health/blob/e9c5e8e2d60c301129ce5369a1b0ae356b3bbf84/pkg/kubernetes/namespace/injected.go#L27